kubernetes/ansible/roles/sunbird-monitoring/templates/additional-scrape-configs.yaml

```diff
@@ -97,4 +97,24 @@ scrapeconfig:
         target_label: instance
       - target_label: __address__
         replacement: blackbox-exporter-prometheus-blackbox-exporter:9115
-  {% endfor %}
+        {% endfor %}
+
+  {% for item in service_health_checks %}
```

Comment:
from where this service _health_checks variable coming ?

Reply:
@rjshrjndrn From here https://github.com/Kaali09/sunbird-devops/blob/release-3.6.0/kubernetes/ansible/roles/sunbird-monitoring/defaults/main.yml#L218